homeassistant/components/proxmoxve/__init__.py

```diff
@@ -62,6 +68,9 @@
                         vol.Optional(
                             CONF_VERIFY_SSL, default=DEFAULT_VERIFY_SSL
                         ): cv.boolean,
+                        vol.Optional(
+                            CONF_SCAN_INTERVAL, default=DEFAULT_SCAN_INTERVAL
```

Comment:
My impression is that there is some special handling for SCAN_INTERVAL if set as a default variable: https://developers.home-assistant.io/docs/integration_fetching_data/#separate-polling-for-each-individual-entity -- so then having it explicitly i the config is not needed. (I may have this wrong.)

Reply:
> Would you be up for splitting up this PR to make review easier / faster? I would recommend sending one platform at a time to make it easier to have a focused review (and i'd be happy to look)

There are also other changes like style improvements or convention updates that can be split out into separate very quick to review PRs, so that would also be nice to do.

Reply:
We no longer accept additions or changes to the yaml config of integrations. See: https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md

This integration needs to be refactored to use a config flow and config entries if the config needs changing.

https://developers.home-assistant.io/docs/config_entries_config_flow_handler

Reply:
good point, that too... but also, `SCAN_INTERVAL` is implicitly defined already right,  so this is not event needed anyway, right?

Reply:
To change the scan interval in the coordinator an option would be needed. But we don't allow that anymore since all integrations that support config entries allow turning off automatic updates and then the user can automate updates using the service `homeassistant.update_entity`.

Reply:
> To change the scan interval in the coordinator an option would be needed. But we don't allow that anymore since all integrations that support config entries allow turning off automatic updates and then the user can automate updates using the service `homeassistant.update_entity`.

Understood - will remove.